src/SFA.DAS.LevyTransferMatching/Application/Services/LevyTransferMatchingService.cs

```diff
@@ -110,6 +110,11 @@ public async Task<ApiResponse<AcceptFundingRequest>> AcceptFunding(AcceptFunding
             return await _levyTransferMatchingApiClient.PostWithResponseCode<AcceptFundingRequest>(request);
         }
 
+        public async Task WithdrawApplication(WithdrawApplicationRequest request, CancellationToken cancellationToken = default)
+        {
+            await _levyTransferMatchingApiClient.PostWithResponseCode<WithdrawApplicationRequest>(request);
```

Comment:
See above, we could return the `ApiResponse<WithdrawApplicationRequest>`, so that it can be used in status checking up the chain.